lib/apiScenario/gen/testScenarioGenerator.ts

```diff
@@ -1,4 +1,4 @@
-import { join as pathJoin, relative as pathRelative, dirname } from "path";
```

Comment:
What's the difference between testScenarioGenerator and apiScenarioGenerator?

Reply:
As above